web/web/models/meter.py

```diff
@@ -176,8 +176,12 @@ def get_interval_coverage(self, obj):
         return MeterInterval.get_interval_coverage(coverage)
 
     def get_user_id(self, obj):
-        if obj.service_location.address.user:
-            return obj.service_location.address.user
+        users = obj.service_location.address.users
+        if users:
+            user_ids = []
+            for user in users:
+                user_ids.append(vars(user)["id"])
+            return user_ids
         return
```

Comment:
Python will automatically return None without the explicit return statement at the end. It might be more beneficial to return an empty user_ids list instead of None if there are no users associated with that meter?

Reply:
done